### PR TITLE
fix(opentracing): default to type entry when no parent is referenced

### PR DESCRIPTION
**Pull request by basti1302**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+- Update to shimmer@1.2.1.
```
Comment (basti1302) [inline CHANGELOG.md:4]: This is unrelated to this PR but I forgot to add it in a previous PR :-/

File: packages/core/src/tracing/opentracing/Span.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ function Span(tracer, name, fields) {
     data: {
       service: serviceName,
       sdk: {
-        type: 'local',
+        type: 'entry',
```
Comment (willianpc) [inline packages/core/src/tracing/opentracing/Span.js:56]: Would it work if we did `type: parentContext ? 'local' : 'entry'`?
It's sort of personal preference, but on the other hand, seems easier to read, as one doesn't need to read the code further to realize that the `type` may change based on whether `parentContext` exists.
But again, this is mostly personal preference.

**Review by willianpc — APPROVED** [review]
LGTM.
I added a small comment with a personal preference, so feel free to do it or not.
Cheers!

File: packages/core/src/tracing/opentracing/Span.js (hunk 2 above)
Comment (basti1302) [reply packages/core/src/tracing/opentracing/Span.js:56]: Absolutely, good call.

**Pull request merged**